HealthcareSystem/Backend/Model/Users/Feedback.cs

```diff
@@ -5,19 +5,28 @@
  ***********************************************************************/
 
 using System;
+using System.Diagnostics;
+using System.Drawing.Printing;
 
 namespace Model.Users
 {
-   public class Feedback
-   {
-        public Stars Grade { get; set; }
+    public class Feedback
+    {
+        public int Id { get; set; }
         public string Comment { get; set; }
+        public bool IsAnonymous { get; set; }
```

Comment:
Is this field necessary? I feel like having a null Commentator property should be enough to make a Feedback anonymous. 
I would consider keeping the IsAnonymous property for convenience, but only the getter and calculate it based on Commentator.